src/radio.cpp

```diff
@@ -38,7 +38,9 @@
 
 zmq::radio_t::radio_t (class ctx_t *parent_, uint32_t tid_, int sid_) :
     socket_base_t (parent_, tid_, sid_, true),
-    _lossy (true)
+    _lossy (true),
+    _notify (false),
+    _pending_notifs ()
```

Comment:
gcc wants the initialisations in the same order as the declarations (it's very annoying):

```
../src/radio.hpp: In constructor ‘zmq::radio_t::radio_t(zmq::ctx_t*, uint32_t, int)’:
../src/radio.hpp:85:10: error: ‘zmq::radio_t::_notify’ will be initialized after [-Werror=reorder]
     bool _notify;
          ^
../src/radio.hpp:80:23: error:   ‘std::deque<zmq::msg_t> zmq::radio_t::_pending_notifs’ [-Werror=reorder]
     std::deque<msg_t> _pending_notifs;
                       ^
../src/radio.cpp:39:1: error:   when initialized here [-Werror=reorder]
 zmq::radio_t::radio_t (class ctx_t *parent_, uint32_t tid_, int sid_) :
```